doc/source/releases/24.0_announce_user.rst

```diff
@@ -0,0 +1,196 @@
+
+===========================================================
+24.0 Galaxy Release (March 2024)
+===========================================================
+
+.. include:: _header.rst
+
+Highlights
+===========================================================
+
+New Workflow List and Card View
+-----------------------------------------------------------
+
+As part of our ongoing efforts to enhance user experience, we are proud to announce several new improvements to the workflows user interface, including workflow list view, workflow card view and workflow quick view. These improvements also include new tabs for "My workflows" (owned by the user), "Shared with me" (shared by other users) and "Published workflows" (publicly accessible workflows).
```

Comment:
Also consider highlighting the ability to embed a preview of a workflow, this will be very user-impacting.

![image](https://github.com/galaxyproject/galaxy/assets/458683/6e5f2c91-df96-4b84-bc08-fc09ea7cbada)

Reply:
I thought it was added in 23.2, but I might be confusing it with a similar feature. I can't find the  specific PR introducing this feature for 24.0. That said, we can certainly include this as a separate bullet point or section with or without a screenshot.